tests/unit/test_cli/test_core_commands.py

```diff
@@ -278,6 +280,58 @@ def test_run_command_debug(
     assert os.getenv("LITESTAR_DEBUG") == "1"
 
 
+@pytest.mark.usefixtures("mock_uvicorn_run", "unset_env")
+def test_run_command_quiet_console(
+    app_file: Path, runner: CliRunner, monkeypatch: MonkeyPatch, create_app_file: CreateAppFileFixture
+) -> None:
+    console = Console(file=io.StringIO())
+    monkeypatch.setattr(_utils, "console", console)
+
+    path = create_app_file("_create_app_with_path.py", content=CREATE_APP_FILE_CONTENT)
+    app_path = f"{path.stem}:create_app"
+    monkeypatch.delenv("LITESTAR_QUIET_CONSOLE", raising=False)
+    result = runner.invoke(cli_command, ["--app", app_path, "run"])
+    assert result.exit_code == 0
+    normal_output = console.file.getvalue()  # type: ignore[attr-defined]
+    assert "Using Litestar from env:" in normal_output
+    assert "Starting server process" in result.stdout
+    del result
+    console = Console(file=io.StringIO())
+    monkeypatch.setattr(_utils, "console", console)
+    monkeypatch.setenv("LITESTAR_QUIET_CONSOLE", "1")
+    assert os.getenv("LITESTAR_QUIET_CONSOLE") == "1"
+    result = runner.invoke(cli_command, ["--app", app_path, "run"])
+    assert result.exit_code == 0
+    quiet_output = console.file.getvalue()  # type: ignore[attr-defined]
+    assert "Starting server process" not in result.stdout
+    assert "Using Litestar from env:" not in quiet_output
```

Comment:
```suggestion
    assert "Using Litestar from env:" not in quiet_output
```
if changed in the above `_utils.py`